core/server/helpers/excerpt.js

```diff
@@ -12,6 +12,24 @@ var hbs = require('express-hbs'),
 function excerpt(options) {
     var truncateOptions = (options || {}).hash || {};
 
+    try {
+        truncateOptions.append = options.hash.append;
+    } catch (e) {
+        // do nothing
+    }
+
+    try {
+        truncateOptions.roundSentence = options.hash.roundSentence;
+    } catch (e) {
+        // do nothing!
+    }
+
+    try {
+        truncateOptions.stripTags = options.hash.stripTags;
+    } catch (e) {
+        // do nothing AGAIN!!!!!!! WHOOOOOO HOOOOOOO!!!!
+    }
+
     truncateOptions = _.pick(truncateOptions, ['words', 'characters']);
```

Comment:
All of the append/roundSentence/stripTags properties should be ignored because of this line.